docs/docs/add-404-page.md

```diff
@@ -1,28 +1,28 @@
 ---
-title: "Adding a 404 Page"
+title: "Agregar una página 404"
 ---
 
-To create a 404 page create a page whose path matches the regex
-`^\/?404\/?$` (`/404/`, `/404`, `404/` or `404`). Most often you'll want to create a React component page at
+Para crear una página 404, cree una página cuya ruta coincida con la expresión regular
+`^\/?404\/?$` (`/404/`, `/404`, `404/` or `404`). La mayoría de las veces querrá crear una página de componentes React en
 `src/pages/404.js`.
 
-Gatsby ensures that your 404 page is built as `404.html` as many static hosting
-platforms default to using this as your 404 error page. If you're hosting your
-site another way, you'll need to set up a custom rule to serve this file for 404
-errors.
+Gatsby asegura que su página 404 se construya como `404.html` tantos hosting estático
+de forma predeterminada para usar esto como su página de error 404. Si estás alojando tu
+de otra manera, deberá configurar una regla personalizada para servir este archivo para 404
+errores.
 
-Because Gatsby creates this page for you by default, there is no need to configure
-it in your `gatsby-node.js` file.
+Como Gatsby crea esta página de forma predeterminada, no es necesario configurar
+en su `gatsby-node.js` file.
```

Comment:
For example here, the translation is weird, the idea would be for line 14 to end with "configurarlo" and then line 15 would be "en tu archivo `gatsby-node.js`".